app/components/Dashboard/ChartsTab/index.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
+import { bindActionCreators } from 'redux';
```

Comment:
could you remove this import?

Reply:
Whoops, forgot to take that out. Yeah one sec